metrics/github/prs.py

```diff
@@ -1,3 +1,23 @@
+def drop_archived_prs(prs, key="created"):
+    """
+    Drop PRs where the given key happened before the repo's archival date
+
+    By default this removes PRs opened after the point at which a repo was
+    archived, but can be configured for other checks, eg merging.
+    """
+
+    def predicate(pr, key):
+        if not pr["repo_archived_at"]:
+            return True
+
+        if pr[key] < pr["repo_archived_at"]:
```

Comment:
I don't think this can be right. It's keeping all PRs that were created/merged before the repo was archived, which will be all of them. Don't we need instead to compare the archive date with the "current" date (where "current" scans through time in back-filling)?

Reply:
_argh_, yep, you're completely right